debugger/src/debug_adapter.rs

```diff
@@ -519,13 +506,26 @@ impl<P: ProtocolAdapter> DebugAdapter<P> {
 
         if let Some(requested_breakpoints) = args.breakpoints.as_ref() {
             for bp in requested_breakpoints {
-                // Try to find source code location
+                // Some overrides to improve breakpoint accuracy when `DebugInfo::get_breakpoint_location()` has to select the best from multiple options
+                let breakpoint_line = if self.lines_start_at_1 && bp.line.is_zero() {
+                    1
+                } else {
+                    bp.line as u64
+                };
+                let breakpoint_column = if self.columns_start_at_1
+                    && (bp.column.is_none() || bp.column.unwrap_or(0) == 0)
+                {
+                    Some(1)
+                } else {
+                    bp.column.map(|c| c as u64)
+                };
```

Comment:
Since we are resolving breakpoints against DWARF references, I was working on their definition in Section 2.14, i.e. columns start at 1 and the value of 0 indicates no column specified. The same things goes for line numbers. 

I will update the code so that incoming (from DAP) values of `None` becomes `1` (that is the way I have it now), and that other values are either incremented by 1 or used as is, depending on if VSCode uses 0 based values or not (currently I only translate the first position value)